variables.tf

```diff
@@ -198,10 +198,10 @@ variable "cluster_delete_timeout" {
   default     = "15m"
 }
 
-variable "local_exec_interpreter" {
-  description = "Command to run for local-exec resources. Must be a shell-style interpreter. If you are on Windows Git Bash is a good choice."
-  type        = list(string)
-  default     = ["/bin/sh", "-c"]
+variable "wait_for_cluster_cmd" {
+  description = "Custom local-exec command to execute for determining if the eks cluster is healthy. Cluster endpoint will be available as an ENDPOINT environment variable"
```

Comment:
Small thing but can you change this to `will be available as an environment variable called ENDPOINT`? Otherwise it sounds like "ENDPOINT" is a special type of env var or something.

Reply:
@sanjeevgiri @max-rocket-internet , i am using the code but still i get an error "Error: Error running command 'until curl -k -s $ENDPOINT/healthz >/dev/null; do sleep 4; done': exit status 1. Output: The system cannot find the path specified." unable to get through and create configmap. can any one of you please help me

Reply:
Hello @sveerabathini, there was a another PR that included a variable local_exec_interpreter. This interpreter uses /bin/sh as the default interpreter. If you are running windows it is likely you will not have a shell installed in that path. If you set the default interpreter value to empty list ([]), then terraform should be able to find a suitable interpreter to use based on the client OS and version (likely power shell for windows). 

I am using git bash, so I added the git bash bin to my windows path environment variable, set the wait_for_cluster_interpreter to []. 

Hope this helps.